plugin/builtin/replacementtransformer/ReplacementTransformer_test.go

```diff
@@ -112,6 +112,68 @@ spec:
 `)
 }
 
+func TestReplacementTransformerFromPathMultiple(t *testing.T) {
```

Comment:
Could you also add a test to [api/krusty/replacementtransformer_test.go](https://github.com/kubernetes-sigs/kustomize/blob/master/api/krusty/replacementtransformer_test.go) so that we can test e2e behavior?

Reply:
Thank you, I've added a test for e2e.